src/main/java/org/jitsi/jigasi/TranscriptionGatewaySession.java

```diff
@@ -184,6 +184,11 @@ public MediaDevice getDefaultDevice(MediaType mediaType,
             {
                 welcomeMessage.append(promise.getDescription());
             }
+
+            if(promise.wantsAudioRecording())
```

Comment:
`wantsAudioRecording` seems superfluous -- you can remove it and just call `giveAudioMixerMediaDevice` (which I would suggest renaming to `setMediaDevice`) without any functional change. I'd prefer it this way, because it is simpler.